src/pybind/mgr/dashboard/controllers/orchestrator.py

```diff
@@ -2,7 +2,11 @@
 from __future__ import absolute_import
 
 import cherrypy
-from ceph.deployment.drive_group import DriveGroupSpec, DriveGroupValidationError
+
+try:
```

Comment:
is this change related to this PR?

Reply:
No, but i got lint errors without this change.